library/src/main/cpp/decoder_heif.cpp

```diff
@@ -29,59 +26,138 @@ ImageInfo HeifDecoder::parseInfo() {
 
   uint32_t imageWidth = handle.get_width();
   uint32_t imageHeight = handle.get_height();
-  Rect bounds = { .x = 0, .y = 0, .width = imageWidth, .height = imageHeight };
+  Rect bounds = {.x = 0, .y = 0, .width = imageWidth, .height = imageHeight};
```

Comment:
This would be easier to review if it didn't include all of the minor formatting changes. I'd be happy to just merge all of the formatting changes as a separate PR first.

Reply:
Done.